npm-shrinkwrap.json

```diff
@@ -5570,11 +5570,6 @@
       "resolved": "https://registry.npmjs.org/read/-/read-1.0.7.tgz",
       "dev": true
     },
-    "read-chunk": {
-      "version": "2.0.0",
-      "from": "read-chunk@>=2.0.0 <3.0.0",
-      "resolved": "https://registry.npmjs.org/read-chunk/-/read-chunk-2.0.0.tgz"
-    },
```

Comment:
There is also `pify`, but there are other modules still depending on it, which is the reason why it still doesn't go away. Luckily more stuff will start getting removed once we get rid of more packages.